tools/schema_loader.cc

```diff
@@ -286,6 +320,28 @@ std::vector<schema_ptr> do_load_schemas(db::config& cfg, std::string_view schema
                 schema = b.build();
             }
             real_db.tables.emplace_back(dd_impl, dd_impl.unwrap(ks), std::move(schema), true);
+        } else if (auto p = dynamic_cast<cql3::statements::create_view_statement*>(statement)) {
+            auto&& [view, warnings] = p->prepare_view(db);
+            auto it = std::find_if(real_db.tables.begin(), real_db.tables.end(), [&] (const table& t) { return t.schema->ks_name() == view->ks_name() && t.schema->cf_name() == view->cf_name(); });
+            if (it != real_db.tables.end()) {
+                continue; // view already exists
```

Comment:
I don't know how to handle these cases in the sstable tool.
If you load the wrong schema, the tool will fail to process the sstable.
In the worst case, it crashes (because we handle mismatching schemas very poorly). This is not the end of the world, this is just a tool after all.